heron/healthmgr/src/java/com/twitter/heron/healthmgr/HealthManager.java

```diff
@@ -173,6 +224,18 @@ public static void main(String[] args) throws Exception {
     }
   }
 
+  private static boolean hasOption(CommandLine cmd, CliArgs argName) {
```

Comment:
If these functions include a single line simple body, I would suggest calling its body directly. I do not see the necessity of wrapping it in function.